mavros/src/plugins/actuator_control.cpp

```diff
@@ -17,6 +17,7 @@
 #include <mavros/mavros_plugin.h>
 
 #include <mavros_msgs/ActuatorControl.h>
+#include <mavros_msgs/ActuatorControlTarget.h>
```

Comment:
I telling about ROS message, not mavlink. And mavlink has `SET_ACTUATOR_CONTROL_TARGET`. No reason to replace existing msg.